src/content/docs/guides/annotate-in-excel.mdx

```diff
@@ -2,11 +2,30 @@
 title: Annotate in Excel
 authors:
   - kevin-frey
+  - patrick-blume
 sidebar:
+  order: 1
   badge:
-    text: beta
+    text: new
     variant: caution
-draft: true
 ---
 
-Lorem ipsum
\ No newline at end of file
+import { Steps } from '@astrojs/starlight/components';
+
+With the help of an [Excel-Add-In](https://learn.microsoft.com/de-de/office/dev/add-ins/excel/excel-add-ins-overview) you can manage your data, utilizing the [ARC](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/core-concepts/arc/) standard, directly in excel, without the necessity to upload it anywhere. 
+The functionality the excel-add-in provides are similar to [Swate](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/swate/) and the differences will be explained in the different sections of swate.
+The excel-add-in will soon be published in the official [Microsoft Excel-Add-Ins Store](https://appsource.microsoft.com/en-us/marketplace/apps?product=excel). 
+
+## Local installation
```

Comment:
must mention online installation as currently recommended workflow. Please use starlight tabs for this

Reply:
fixed